tests/pytests/test_geo.py

```diff
@@ -54,10 +54,10 @@ def testGeoDistanceSimple(env):
   env.expect('FT.SEARCH', 'idx', '@location:[1.23 86 10 km]').equal([0])
   # test profile
   env.cmd('FT.CONFIG', 'SET', '_PRINT_PROFILE_CLOCK', 'false')
-  res = ['Iterators profile', ['Type', 'GEO', 'Term', '1.23,4.55 - 1.24,4.56', 'Counter', 4, 'Size', 4]]
+  res = ['Type', 'GEO', 'Term', '1.23,4.55 - 1.24,4.56', 'Counter', 4, 'Size', 4]
 
   act_res = env.cmd('FT.PROFILE', 'idx', 'SEARCH', 'QUERY', '@location:[1.23 4.56 10 km]', 'nocontent')
-  env.assertEqual(act_res[1][4], res)
+  env.assertEqual(act_res[1][1][0][3], res)
```

Comment:
Consider making a generalizing API such that we don't use these (ugly) indexes every Profile test

Reply:
The more generalized tests the better IMHO, so that the next person that makes a change won't have to modify all these tests.

Reply:
Not critical for this PR. It's something we should add for more commands in general in our test-suite, it's just a good opportunity to do it for `FT.PROFILE`.. Your call